BMM.UI.iOS/Application/CustomViews/BmmSliderView.cs

```diff
@@ -21,6 +23,18 @@ public BmmSliderView(CGRect frame) : base(frame)
         {
         }
         
-        public override bool BeginTracking(UITouch uitouch, UIEvent? uievent) => true;
+        public IMvxCommand<float> SliderTouchedCommand { get; set; }
+
+        public override void TouchesEnded(NSSet touches, UIEvent evt)
+        {
+            if (touches.Any())
+            {
+                var locationInView = ((UITouch)touches.First()).LocationInView(this);
+                var touchedValue = (locationInView.X / Frame.Width) * MaxValue;
+                SliderTouchedCommand?.Execute((float)touchedValue);
```

Comment:
why is this casted to float and then in the next step to int? Can this be simplified?

Reply:
Good catch. Changed command parameter to long.